docs/source/logging/logging.md

```diff
@@ -1,54 +1,73 @@
 
-# Default framework-side logging configuration
+By default, Python only shows logging messages at level WARNING and above. Kedro's logging configuration specifies that INFO level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a kedro run.
 
-Kedro's [default logging configuration](https://github.com/kedro-org/kedro/blob/main/kedro/framework/project/default_logging.yml) defines a handler called `rich` that uses the [Rich logging handler](https://rich.readthedocs.io/en/stable/logging.html) to format messages. We also use the [Rich traceback handler](https://rich.readthedocs.io/en/stable/traceback.html) to render exceptions.
+## Perform logging in your project
+To perform logging in your own code (e.g. in a node), you are advised to do as follows:
+
+```python
+import logging
 
-By default, Python only shows logging messages at level `WARNING` and above. Kedro's logging configuration specifies that `INFO` level messages from Kedro should also be emitted. This makes it easier to track the progress of your pipeline when you perform a `kedro run`.
+log = logging.getLogger(__name__)
+log.warning("Issue warning")
+log.info("Send information")
+log.debug("Useful information for debugging")
+```
 
-## Project-side logging configuration
+```{note}
+The name of a logger corresponds to a key in the `loggers`  section in `logging.yml` (e.g. `kedro`). See [Python's logging documentation](https://docs.python.org/3/library/logging.html#logger-objects) for more information.
+```
 
-In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/base/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines two further logging handlers:
+You can take advantage of rich's [console markup](https://rich.readthedocs.io/en/stable/markup.html) when enabled in your logging calls:
+```python
+log.error("[bold red blink]Important error message![/]", extra={"markup": True})
+```
+
+### Show DEBUG level messages
+To see your `DEBUG` level message, you can change the level of log messages that you wish to see in `logging.yml`.
+
+```yml
+loggers:
+  kedro:
+    level: INFO
+
+  your_python_pacakge:
+    level: DEBUG  # Change this to DEBUG
+```
+
+By changing the level value to `DEBUG` for the desired logger (e.g., <your_python_package>), you will start seeing `DEBUG` level messages in the log output.
+
+## Customise Logging
+In addition to the `rich` handler defined in Kedro's framework, the [project-side `conf/logging.yml`](https://github.com/kedro-org/kedro/blob/main/kedro/templates/project/%7B%7B%20cookiecutter.repo_name%20%7D%7D/conf/base/logging.yml) defines two further logging handlers:
```

Comment:
`conf/logging.yml` will no longer exist - see: https://github.com/kedro-org/kedro/issues/2281 . Additionally `console` and `info_file_handler` are not included in the default logging setup